middleman_protocol/tests/utils.py

```diff
@@ -0,0 +1,16 @@
+import assertpy
+
+
+def assertpy_bytes_starts_with(string: bytes, string_to_check: bytes):
+    """
+    Custom function which allows testing if given bytes start with other bytes using assertpy.
+    """
+
+    assert isinstance(string, bytes)
+    assert isinstance(string, bytes)
```

Comment:
These asserts look identical. You need only one :)

Reply:
It must have been very late :+1: done